Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["cryptography::cryptocurrencies"]
 
 [dependencies]
 tree_hash = "0.6.0"
-ethereum_ssz = "0.5.0"
+ethereum_ssz = { git = "https://github.com/macladson/ethereum_ssz", branch = "bitvector" }
```

Comment:
Once the PR in `ethereum_ssz` has been merged, we should change this back to the appropriate version